src/Symfony/Component/Form/ResolvedFormType.php

```diff
@@ -92,7 +92,11 @@ public function getTypeExtensions()
      */
     public function createBuilder(FormFactoryInterface $factory, $name, array $options = [])
     {
-        $options = $this->getOptionsResolver()->resolve($options);
+        try {
+            $options = $this->getOptionsResolver()->resolve($options);
+        } catch (\Throwable $e) {
```

Comment:
I would only catch exceptions here that are expected to be thrown by options resolver.

Reply:
Done.